doc/cloudbuild_flowchart.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have included this, so that the flowchart can be edited and updated using this file.

Reply:
Great, thank you.